hack/make.sh

```diff
@@ -255,6 +255,7 @@ main() {
 		rm -fr bundles/$VERSION && mkdir bundles/$VERSION || exit 1
 		echo
 	fi
+	ln -sfT $VERSION bundles/latest
```

Comment:
Thanks to #11112, _you_ can pretty trivially. :+1:

I'm not totally comfortable with the idea of them just being in the `PATH` all the time, since that could cause some serious confusion if we're not careful.